backend/app/generate_badges.py

```diff
@@ -73,6 +77,10 @@ def generate_badges(self, aggregate, folder, index, picture, paper_size):
                                       "font-family:" + self.font_choice)
             content = content.replace("inkscape-font-specification:ubuntu",
                                       "inkscape-font-specification:" + self.font_choice)
+        print(self.font_size)
```

Comment:
Remove this maybe ?

Reply:
Yaa, it is for debugging. Will remove it.